### PR TITLE
Create team-members-t1-2024.md

### PR DESCRIPTION
**Pull request by CchristiNana**
# Description

_Please include a summary of the change and which issue is fixed. Please also include relevant
motivation and context. List any dependencies that are required for this change._

Fixes # (issue)

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Documentation (update or new)

## How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration_

## Testing Checklist

- [ ] Tested in latest Chrome
- [ ] Tested in latest Safari
- [ ] Tested in latest Firefox

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have requested a review from ... on the Pull Request

### CODE REVIEW

**Comment by CchristiNana** [comment]
[Update Team Member T1 2024 (Documentation Website)](https://trello.com/c/bNiiYJLQ/172-update-team-member-t1-2024-documentation-website)

File: src/content/docs/teams-and-leadership/team-members-t1-2024.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+| Daniel Piscioneri                        | Team Member        | Work on Trello tasks for various projects   |
+
+## SplashKit
+
+| Project Member           | Role         | Responsibilities                     |
+| ------------------------ | ------------ | ------------------------------------ |
+| Aditya Parmar            | Product Lead | Team management, Code, Documentation |
+| Sean Boettger            | Product Lead | Team management, Code, Documentation |
+| Satyam Raina             | Team member  | Code, Documentation                  |
+| Hayden Hughes            | Team member  | Code, Documentation                  |
```
Comment (omckeon) [inline src/content/docs/teams-and-leadership/team-members-t1-2024.md:45]: Update this name to be: Hayley Hughes

File: src/content/docs/teams-and-leadership/team-members-t1-2024.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+| Wenxuan Song             | Team member  | Code, Documentation                  |
+| William Robert Saunders  | Team member  | Code, Documentation                  |
+| Oscar Harris             | Team member  | Code, Documentation                  |
+| Jonathan James Tynan     | Team member  | Code, Documentation                  |
+| Jake Andrew Oxley        | Team member  | Code, Documentation                  |
+| Sharvani Kandala         | Team member  | Code, Documentation                  |
+| Kartik Kaushik           | Team member  | Code, Documentation                  |
+| Thomas James Shanahan    | Team member  | Code, Documentation                  |
+| Ka Chun Lam              | Team member  | Code, Documentation                  |
+| James Stuart Balsillie   | Team member  | Code, Documentation                  |
```
Comment (omckeon) [inline src/content/docs/teams-and-leadership/team-members-t1-2024.md:64]: Update this name to be: Jess Balsillie

**Review by omckeon — CHANGES_REQUESTED** [review]
In the SplashKit Team:
Please update a few names that are incorrect.

Rest looks good.

**Review by omckeon — APPROVED** [review]
Much better :) Thanks!

**Pull request merged**